modules/apps/frontend-theme/frontend-theme-favicon-servlet/src/main/java/com/liferay/frontend/theme/favicon/servlet/internal/FaviconServlet.java

```diff
@@ -51,21 +51,14 @@ protected void doGet(
 		ThemeDisplay themeDisplay = (ThemeDisplay)request.getAttribute(
 			WebKeys.THEME_DISPLAY);
 
-		if (themeDisplay != null) {
-			response.sendRedirect(themeDisplay.getPathImage() + "/favicon.ico");
```

Comment:
Maybe we can leave this condition in the background when we do not receive `CURRENT_URL`, for example. What do you think?

Reply:
I apologize, I meant to remove ThemeDisplay entirely but missed this. At this point I am not sure if there is a way to get the themeDisplay information into the servlet. We could readd it to show what is intended to happen I suppose.

Reply:
No, it would always return `null`. We had `LayoutSet` which would be the default theme.

Reply:
oh sorry for the confusion, actually what I wanted to say is `layoutSet` that we got the default theme.

Reply:
No worries at all :) Yeah you are right.